### ISSUE TITLE
get error while training

### ISSUE THREAD

**Issue by seamanj** [akanazawa/human_dynamics#42]
Dear author

when I was training, I got the following errors. do you think it is fatal? what should I do with these? thanks

`
2019-12-04 14:20:13.054281: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2019-12-04 14:20:13.090140: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2208000000 Hz
2019-12-04 14:20:13.091755: I tensorflow/compiler/xla/service/service.cc:150] XLA service 0x562da46dafc0 executing computations on platform Host. Devices:
2019-12-04 14:20:13.091783: I tensorflow/compiler/xla/service/service.cc:158]   StreamExecutor device (0): <undefined>, <undefined>
WARNING:tensorflow:From /home/seamanj/Software/anaconda3/envs/human_dynamics/lib/python3.7/site-packages/tensorflow/python/training/saver.py:1266: checkpoint_exists (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:
Use standard file APIs to check for files with this prefix.
loading from /home/seamanj/Software/human_dynamics/models/hmr_noS5.ckpt-642561
2019-12-04 14:20:27.351456: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node strided_slice_66. Error: Pack node (smpl_main_4/stack_1) axis attribute is out of bounds: 2
2019-12-04 14:20:27.351497: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node strided_slice_68. Error: Pack node (smpl_main_5/stack_1) axis attribute is out of bounds: 2
2019-12-04 14:20:27.353487: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node batch_orth_proj_idrot_1/strided_slice. Error: Pack node (smpl_main_3/stack_1) axis attribute is out of bounds: 2
2019-12-04 14:20:27.354405: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node strided_slice_32. Error: Pack node (smpl_main_1/stack_1) axis attribute is out of bounds: 2
2019-12-04 14:20:27.354423: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node strided_slice_34. Error: Pack node (smpl_main_2/stack_1) axis attribute is out of bounds: 2
2019-12-04 14:20:27.356630: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node batch_orth_proj_idrot/strided_slice. Error: Pack node (smpl_main/stack_1) axis attribute is out of bounds: 2
2019-12-04 14:20:32.819601: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node strided_slice_66. Error: Pack node (smpl_main_4/stack_1) axis attribute is out of bounds: 2
2019-12-04 14:20:32.819636: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node strided_slice_68. Error: Pack node (smpl_main_5/stack_1) axis attribute is out of bounds: 2
2019-12-04 14:20:32.821490: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node batch_orth_proj_idrot_1/strided_slice. Error: Pack node (smpl_main_3/stack_1) axis attribute is out of bounds: 2
2019-12-04 14:20:32.822380: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node strided_slice_32. Error: Pack node (smpl_main_1/stack_1) axis attribute is out of bounds: 2
2019-12-04 14:20:32.822398: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node strided_slice_34. Error: Pack node (smpl_main_2/stack_1) axis attribute is out of bounds: 2
2019-12-04 14:20:32.824279: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node batch_orth_proj_idrot/strided_slice. Error: Pack node (smpl_main/stack_1) axis attribute is out of bounds: 2
2019-12-04 14:20:39.175761: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node strided_slice_66. Error: Pack node (smpl_main_4/stack_1) axis attribute is out of bounds: 2
2019-12-04 14:20:39.175816: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node strided_slice_68. Error: Pack node (smpl_main_5/stack_1) axis attribute is out of bounds: 2
2019-12-04 14:20:39.180563: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node batch_orth_proj_idrot_1/strided_slice. Error: Pack node (smpl_main_3/stack_1) axis attribute is out of bounds: 2
2019-12-04 14:20:39.182307: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node strided_slice_32. Error: Pack node (smpl_main_1/stack_1) axis attribute is out of bounds: 2
2019-12-04 14:20:39.182335: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node strided_slice_34. Error: Pack node (smpl_main_2/stack_1) axis attribute is out of bounds: 2
2019-12-04 14:20:39.185923: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node batch_orth_proj_idrot/strided_slice. Error: Pack node (smpl_main/stack_1) axis attribute is out of bounds: 2
2019-12-04 14:21:11.057179: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node strided_slice_66. Error: Pack node (smpl_main_4/stack_1) axis attribute is out of bounds: 2
2019-12-04 14:21:11.057221: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node strided_slice_68. Error: Pack node (smpl_main_5/stack_1) axis attribute is out of bounds: 2
2019-12-04 14:21:11.060505: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node batch_orth_proj_idrot_1/strided_slice. Error: Pack node (smpl_main_3/stack_1) axis attribute is out of bounds: 2
2019-12-04 14:21:11.062146: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node strided_slice_32. Error: Pack node (smpl_main_1/stack_1) axis attribute is out of bounds: 2
2019-12-04 14:21:11.062165: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node strided_slice_34. Error: Pack node (smpl_main_2/stack_1) axis attribute is out of bounds: 2
2019-12-04 14:21:11.064580: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node batch_orth_proj_idrot/strided_slice. Error: Pack node (smpl_main/stack_1) axis attribute is out of bounds: 2
`

**Comment by jasonyzhang** [comment]
Some of these errors do look fatal.

Please check that you are using CUDA 9.0 and tensorflow version 1.8. The tensorflow API in particular has changed a bit, so you may need to make modifications to the code if you intend to use a newer version.
